service/src/main/resources/quips/complibot/winsults.json

```diff
@@ -38,5 +38,45 @@
     "usageRules":[
       
     ]
+  },
+  {
+    "quipGroup":"BET_ON_YOU",
+    "text":"I wouldn't bet on you to do well in life. It's technically not gambling if it's a sure thing.",
```

Comment:
"I wouldn't bet on you to do well in life. Because technically it's not gambling if it's a sure thing." ?